gateway-js/src/executeQueryPlan.ts

```diff
@@ -501,6 +501,21 @@ async function executeFetch(
         downstreamServiceError(error, fetch.serviceName, errorPathHelper),
       );
       context.errors.push(...errors);
+
+      if (!response.extensions?.ftv1) {
+        const errorPaths = errors.map((error) => ({
```

Comment:
does this work? my comment at https://github.com/apollographql/apollo-server/pull/7136#discussion_r1042967523 suggests that this should be looking at `response.errors` instead? haven't re-verified that though.

Reply:
Good catch! More complex example with `_entities` fetches revealed this. Now we have better test cases too.